operator/pom.xml

```diff
@@ -25,10 +25,13 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <awaitility.version>4.2.0</awaitility.version>
-        <kubernetes-client-java-gen.version>6.7.2</kubernetes-client-java-gen.version>
-        <sundrio.version>0.95.0</sundrio.version>
+        <kubernetes-client-java-gen.version>6.8.1</kubernetes-client-java-gen.version>
         <assertj.core.version>3.24.2</assertj.core.version>
-        <quarkus.operator.sdk.version>5.1.4</quarkus.operator.sdk.version>
+        <bouncycastle.version>1.70</bouncycastle.version>
+        <!-- ATTENTION! The Quarkus version needs to be aligned with the Quarkus Operator SDK -->
+        <!-- TODO: possibly stabilize on something shared with the rest of the repo -->
+        <quarkus.version>3.2.0.Final</quarkus.version>
```

Comment:
In this module the Quarkus version should be aligned with the one used by the Operator SDK.
@carlesarnal any specific reason for not bumping to 3.2 even in the rest of the project?

Reply:
No reason at all, 3.0.2.Final was just the one available when I worked on the migration.

Reply:
let me merge this one first, and we can eventually align 👍 is dependabot configured to bump quarkus core deps?